scripts/getDependencies.pl

```diff
@@ -128,6 +128,11 @@
 		url => ' https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.16.100/org.eclipse.osgi-3.16.100.jar',
 		fname => 'org.eclipse.osgi-3.16.100.jar',
 		sha1 => '7ddb312f386b799d6e004d193a01c50169bf69f3'
+	},
+	jython => {
+		url => ' https://repo1.maven.org/maven2/org/python/jython-standalone/2.7.2/jython-standalone-2.7.2.jar',
+		fname => 'jython-standalone-2.7.2.jar',
```

Comment:
Could we remove the version in the name? In this way, if we upgrade the version, we do not need to update the related tests in other repos.